contracts/LifCrowdsale.sol

```diff
@@ -255,9 +255,10 @@ contract LifCrowdsale is Ownable, Pausable {
 
       // TODO: create the market maker with a start block that equals one month after crowdsale ends
       LifMarketMaker newMarketMaker = new LifMarketMaker(
-        address(token), block.number.add(10), 20, marketMakerPeriods, foundationWallet
+        address(token), block.timestamp.add(10), 30 days, marketMakerPeriods, foundationWallet
       );
       newMarketMaker.fund.value(mmFundBalance)();
+      newMarketMaker.transferOwnership(foundationWallet);
```

Comment:
can we do this in a separate commit or even PR? It'd also be nice to add a test for this, for example in the `finalizeCrowdsale` test command we could assert who is the owner

Reply:
I will do this on a separate commit, and assert the owner.

Reply:
I don't see this change in a separate commit yet, also I couldn't find the assertion. Would be great to extract it into a separate PR

Reply:
https://github.com/windingtree/LifToken/pull/170/commits/6009468216dc50346998a680bcb8bd27df6ea87b here  is the commit, but you are right, the assert is missing.

Reply:
I saw that commit, but it's not only about the ownership thing. And I insist: can we move this to a separate PR? We can merge that PR before this one, it shouldn't take much time I think

Reply:
Yes we can, I will need to edit the commit and transferOwnerhsip outside the contract on the simulateCrowdsale function. Leaving only the change on the constructor.

Reply:
I think you can just start a new branch from master, add this line and the assertion in a test, and create a commit with just those changes. Once that's merged, you would merge master here, resolve the conflicts, and once this PR is finished you can rewrite the commits with a rebase

Reply:
ok :disappointed_relieved:

Reply:
Sorry if it sounds like too much work. I don't think it is, and having smaller and focused PRs helps a lot in understanding what the code is doing and why. Not only to who is reviewing the code while writing it but also to future readers of the codebase/commit history. This ownership issue is a big one that we were missing, it deserves its own PR.